dev/gpctl/cmd/clusters-register.go

```diff
@@ -63,21 +63,29 @@ var clustersRegisterCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		if len(name) < 1 && len(request.Name) < 1 {
+		if len(name) < 1 {
```

Comment:
Why changing the pattern here? Now we no longer can load name, region and url from file IIreadC...? :thinking: